tests/integration_tests/data_sources/test_postgres_ds.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import pg8000
+from mindsdb import Predictor
+from mindsdb.libs.data_sources.postgres_ds import PostgresDS
+
+
+def test_postgres_ds():
+    HOST = 'localhost'
+    USER = 'postgres'
+    PASSWORD = ''
+    DBNAME = 'postgres'
+    PORT = 5432
+
+    con = pg8000.connect(database=DBNAME, user=USER, password=PASSWORD,
+                         host=HOST, port=PORT)
+    cur = con.cursor()
+
+    cur.execute('DROP TABLE IF EXISTS test_mindsdb')
+    cur.execute(
+        'CREATE TABLE test_mindsdb(col_1 Text, col_2 Int, col_3 Boolean)')
+    for i in range(0, 200):
+        cur.execute(
+            f'INSERT INTO test_mindsdb VALUES (\'This is tring number {i}\', {i}, {i % 2 == 0})')
```

Comment:
I *think* pg8000 should have an insert syntax where arguments are passed separately *and* if not you can use `"""` instead of `'` to forgo the escaping of `'`... but whatever, minor nitpick, no need to change, just good to remember in the future.

Reply:
I agree, but for tests this will do since we don't need to protect form some injection here